src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamespaceSymbol.cs

```diff
@@ -246,6 +248,8 @@ private int GetQualifiedNameLength()
             return length;
         }
 
+#nullable restore
```

Comment:
This looks unnecessary - I think `restore` effectively means `enable` here and you already have `enable` above.

(Perhaps you meant `#nullable disable` but then you wouldn't need to add annotations below.)